torch/_dynamo/symbolic_convert.py

```diff
@@ -2234,9 +2234,8 @@ def check_inlineable(func):
         except NotImplementedError:
             pass  # closures
 
-        if skipfiles.check(
-            func.get_filename()
-        ) and not skipfiles.is_torch_inline_allowed(func.get_filename()):
+        result = skipfiles.check_verbose(func.get_filename(), allow_torch=True)
```

Comment:
The before and after here are not obviously equivalent to me

Reply:
You are right, they are not identical! But this is intended, since we want to consolidate these rules. I'm trying to make the skip/inline list and check rules logically more organized in this refactor.